Base/QTGUI/qSlicerDarkStyle.cxx

```diff
@@ -41,31 +41,31 @@ QPalette qSlicerDarkStyle::standardPalette()const
 
   // See https://doc.qt.io/qt-5/qpalette.html#ColorRole-enum
   // Central color roles
-  palette.setColor(QPalette::Window, "#353535");
+  palette.setColor(QPalette::Window, "#323232");
   palette.setColor(QPalette::WindowText, Qt::white);
-  palette.setColor(QPalette::Disabled, QPalette::WindowText, "#7f7f7f");
-  palette.setColor(QPalette::Base, "#2d2d2d");
-  palette.setColor(QPalette::Disabled, QPalette::Base, "#323234");
-  palette.setColor(QPalette::AlternateBase, "#424242");
+  palette.setColor(QPalette::Disabled, QPalette::WindowText, "#6d6d6d");
+  palette.setColor(QPalette::Base, "#1e1e1e");
+  palette.setColor(QPalette::Disabled, QPalette::Base, "#2d2d2d");
+  palette.setColor(QPalette::AlternateBase, "#323232");
   palette.setColor(QPalette::ToolTipBase, Qt::white);
-  palette.setColor(QPalette::ToolTipText, "#353535");
+  palette.setColor(QPalette::ToolTipText, "#323232");
   palette.setColor(QPalette::Text, Qt::white);
-  palette.setColor(QPalette::Disabled, QPalette::Text, "#7f7f7f");
-  palette.setColor(QPalette::Button, "#353535");
+  palette.setColor(QPalette::Disabled, QPalette::Text, "#6d6d6d");
+  palette.setColor(QPalette::Button, "#323232");
   palette.setColor(QPalette::ButtonText, Qt::white);
-  palette.setColor(QPalette::Disabled, QPalette::ButtonText, "#7f7f7f");
+  palette.setColor(QPalette::Disabled, QPalette::ButtonText, "#6d6d6d");
   palette.setColor(QPalette::BrightText, Qt::red);
   // Color roles used mostly for 3D bevel and shadow effects.
   palette.setColor(QPalette::Light, "#828284");  // Lighter than Button color.
-  palette.setColor(QPalette::Midlight, "#5c5c5d");  // Between Button and Light.
+  palette.setColor(QPalette::Midlight, "#5a5a5b");  // Between Button and Light.
   palette.setColor(QPalette::Dark, "#232323");  // Darker than Button.
-  palette.setColor(QPalette::Mid, "#2c2c2c");  // Between Button and Dark.
+  palette.setColor(QPalette::Mid, "2b2b2b");  // Between Button and Dark.
```

Comment:
`2b2b2b` -> `#2b2b2b` ?

Reply:
Indeed that is a typo. I will fix that too